packages/cyberstorm/src/components/Dialog/DialogClose.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+import { ReactNode } from "react";
+import * as RadixDialog from "@radix-ui/react-dialog";
+
+export interface DialogCloseProps extends RadixDialog.DialogCloseProps {
+  children?: ReactNode;
+}
+
+/**
+ * Cyberstorm Dialog Close Sub Component
+ */
+export function DialogClose(props: DialogCloseProps) {
+  const { children, ...restOfProps } = props;
+
+  return <RadixDialog.Close {...restOfProps}>{children}</RadixDialog.Close>;
+}
+
+DialogClose.displayName = "DialogClose";
```

Comment:
Not sure if this was discussed and decided we want to have the prefix (e.g. Dialog) in the names of all the subcomponents, but this approach might make the naming cleaner: https://react-typescript-cheatsheet.netlify.app/docs/advanced/misc_concerns/#namespaced-components

Reply:
I removed the "Dialog" prefix, but I couldn't get that thing working that was suggested in the link.

Problem came when I had to export the component through packages/cyberstorm/src/index.

I've added a task to sort out the imports and export as a whole after the MVP. Seems like there's a lot of inconsistencies, which might be more efficient to fix all at ones.